src/components/Auth/UsersCard/UsersCard.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { Portal } from '@rmwc/base';
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import configureStore from '../../../configureStore';
+import { waitForDialogsToOpen } from '../../../test_utils';
+import UsersCard from './UsersCard';
+
+jest.mock('./table/UsersTable');
+jest.mock('./header/AuthFilter');
+
+function setup() {
+  const store = configureStore();
+
+  return render(
+    <>
+      <Provider store={store}>
```

Comment:
Since lots of nested components use connect w/ redux store, it's either that, or mocking many components. 
However some of the components provide DOM elements that participate in the test, so I chose not to mock them.
I can mock and drop the Provider them if you feel strongly

Reply:
I think your suggestion is better, just make sure your tests don't depend on things in the real store, especially not the test data in there right now. Using an isolated fake store is also a valid option here.